//----------------------------------------------------------------------------
// copyright 2025, jessa gillespie, all rights reserved.
//----------------------------------------------------------------------------

syntax = "proto3";

option optimize_for = LITE_RUNTIME;

message JointData_proto
{
	optional uint32 numJoints = 1;
	repeated uint32 poJointEntry = 2;
}

// -- End of File ---
